nodeList.json

```diff
@@ -1,13 +1,16 @@
 {
   "nodeList": [
     {
+      "routerModel": "edgerouter-x-sfp",
```

Comment:
Should this just be collapsed to "model" for both "routerModel" and "antennaModel"